outlook/indico_outlook/calendar.py

```diff
@@ -106,17 +107,35 @@ def _update_calendar_entry(entry, settings):
         if event.is_deleted:
             logger.debug('Ignoring %s for deleted event %s', entry.action.name, entry.event_id)
             return True
-        location = strip_control_chars(event.room_name)
-        description = strip_control_chars(event.description)
-        event_url = event.external_url
+
+        location = (f'{event.room_name} ({event.venue_name})'
+                if event.venue_name and event.room_name
+                else (event.venue_name or event.room_name))
+
+        cal_description = []
+        if event.person_links:
+            speakers = [f'{x.full_name} ({x.affiliation})' if x.affiliation else x.full_name
+                    for x in event.person_links]
+            cal_description.append(f'<p>Speakers: {', '.join(speakers)}</p>')
+        cal_description.append(event.description)
+        if event.external_url:
+            cal_description.append(f'<p><a href="{event.external_url}">{event.external_url}</a></p>')
+
+        # Try to add the VC room URL to the description, if there is one
+        for vc_room in event.vc_room_associations:
+            try:
+                cal_description.append(f'<p><a href="{vc_room.data.url}">{vc_room.data.url}</a></p>')
+            except AttributeError:
+                pass
```

Comment:
`data` is a dict, so this will always go in the `except` block

Reply:
Oops, I got carried away copying from the javascript part.